thriftbp/server.go

```diff
@@ -8,6 +8,7 @@ import (
 
 	"github.com/reddit/baseplate.go"
 	"github.com/reddit/baseplate.go/errorsbp"
+
```

Comment:
nit: this blank line will cause weird sorting issues in the future (e.g. people might be adding another import after `errorsbp`)

Reply:
interesting, I'm not sure what put that here.  It might've been goimports, which is how I got the import added after the find/replace.  Fixed, thanks.